konflux-ci/ui/core/proxy/proxy.yaml

```diff
@@ -29,37 +29,6 @@ spec:
         labelSelector:
           matchLabels:
             app: proxy
-      initContainers:
-      - name: add-sva-token-to-nginx-config
-        image: registry.access.redhat.com/ubi9/ubi@sha256:66233eebd72bb5baa25190d4f55e1dc3fff3a9b77186c1f91a0abdb274452072
-        command:
```

Comment:
this part we will need. it's the token used for impersonation when nginx forwards requests to the kube api.